docs/font-inspector.html

```diff
@@ -231,6 +243,26 @@ <h1>Free Software</h1>
             element.innerHTML = '<dt>' + Object.keys(font.kerningPairs).length + ' Pairs</dt><dd>' + JSON.stringify(font.kerningPairs) + '</dd>';
         }
     }
+
+    if(font.tables.cpal) {
+        let markup = ' ';
+        const cpal = document.getElementById("cpal-table");
+        const dt = cpal.querySelector('dt');
+        if ( dt ) {
+            const palettes = font.palettes.getAll('hexa');
+            if ( palettes.length ) {
+                palettes.forEach((palette, idx) => {
+                    markup += `<dt><strong>↪ Palette ${idx}</strong></dt><dd><div class="color-swatches">`;
+                        palette.forEach(color => {
+                            markup += `<span class="color-swatch" style="background:${color}" title="${color}"></span>`;
+                        });
+                    markup += `</div></dd>`;
+                });
+                cpal.innerHTML += markup;
```

Comment:
`inneHMTL` could be the result of a double .map()

```js
cpal.innerHTML += palettes.map((palette, idx) =>
    `<dt><strong>↪ Palette ${idx}</strong></dt><dd><div class="color-swatches">`+
    palette.map(color => `<span class="color-swatch" style="background:${color}" title="${color}"></span>`).join('')+
    `</div></dd>`);
});
```

Reply:
Nice simplification indeed!

Reply:
Implemented. It actually needed a second `.join('')` at the and for multiple palettes to look right.